app/models/location.rb

```diff
@@ -0,0 +1,15 @@
+class Location < ApplicationRecord
+  acts_as_mappable lat_column_name: :latitude, lng_column_name: :longitude
+
+  belongs_to :locatable, inverse_of: :location, polymorphic: true
+
+  validates :latitude, :longitude, numericality: true, presence: true
+
+  default_scope { includes(:locatable) }
+
+  self.per_page = 10
+
+  def self.transportation_offices_and_weight_scales_by_distance(latitude:, longitude:)
```

Comment:
what does [param]: mean?

Reply:
[Thoughtbot to the rescue!](https://robots.thoughtbot.com/ruby-2-keyword-arguments)

Ruby 2 introduced this style of keyword arguments (as opposed to positional arguments). This method has two required keyword arguments, `latitude` and `longitude`. The `foo:` syntax indicates the argument is required but doesn't set a default value. This could be expressed with defaults as:

```rb
def self.transportation_offices_and_weight_scales_by_distance(latitude: 0, longitude: 0)
```